pkgs/development/python-modules/simplebayes/default.nix

```diff
@@ -22,16 +22,14 @@ buildPythonPackage {
   };
 
   nativeCheckInputs = [
-    nose
     mock
+    unittestCheckHook
```

Comment:
See https://github.com/NixOS/nixpkgs/pull/330001#discussion_r1693332883. (This is still an improvement, though.)